accounts/views_profile.py

```diff
@@ -65,4 +67,95 @@ def profile_update(request):
         else:
             form = ProfileForm(profile.get_populated_fields())
     template_data['form'] = form
-    return render(request, 'ienitk/profile/update.html', template_data)
\ No newline at end of file
+    return render(request, 'ienitk/profile/update.html', template_data)
+
+
+def apply(request):
+    authentication_result = views.authentication_check(request)
+    if authentication_result is not None: return authentication_result
+    template_data = views.parse_session(request, {'form_button': "Submit"})
+
+    account = Account.objects.get(pk=request.user.id)
+
+    stat = Status.objects.filter(user=account).first()
+
+    if stat is None:
+        if request.method == 'POST':
+            form = SIGForm(request.POST)
+            if form.is_valid():
+                i = 0
+                if i == 0:
+                    views.register_SIG(
+                        form.cleaned_data['SigMain1'],
+                        account,
+                        datetime.datetime.now(),
+                        "WR"
+                    )
+                    i = i + 1
+                if i == 1:
+                    views.register_SIG(
+                        form.cleaned_data['SigMain2'],
+                        account,
+                        datetime.datetime.now(),
+                        "WR"
+                    )
+                    i = i + 1
+                if i == 2:
+                    views.register_SIG(
+                        form.cleaned_data['SigAux1'],
+                        account,
+                        datetime.datetime.now(),
+                        "WR"
+                    )
+                    i = i + 1
+                if i == 3:
+                    views.register_SIG(
+                        form.cleaned_data['SigAux2'],
+                        account,
+                        datetime.datetime.now(),
+                        "WR"
+                    )
+                    i = i + 1
+            request.session['alert_success'] = "Successfully registered SIGS with the portal."
+            data = Status.objects.filter(user=account)
+            data1 = []
+            DB_Status = Status.STATUS_TYPES
+            DB_SIG = Status.SIG_TYPES
+            for entry in data:
+                for type in DB_SIG:
+                    if entry.SIG == type[0]:
+                        sig_name = type[1]
+                for status in DB_Status:
+                    if entry.status == status[0]:
+                        sig_status = status[1]
+                data1.append([sig_name, sig_status])
```

Comment:
Please rename `data` and `data1` to meaningful variables which would tell us what that data represents.